soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -3888,6 +3899,34 @@ void DrawRandoEditor(bool& open) {
                 ImGui::BeginChild("ChildMiscWorldSettings", ImVec2(0,-8));
                 ImGui::PushItemWidth(-FLT_MIN);
 
+                //Starting Age
+                //Disabled when Forest is set to Closed
+                bool disableRandoStartingAge = !CVar_GetS32("gRandomizeForest", 0);
```

Comment:
Same as above, does this not disable the option when forest is set to "Closed Deku"?

Reply:
See video above.